README.md

```diff
@@ -83,6 +83,28 @@ You can set up our Nix module for GTK with the following config:
 
 Alternatively, if you are not using our Nix module, you can grab the theme from [nixpkgs/catppuccin-gtk](https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/themes/catppuccin-gtk/default.nix)
 
+## Flatpak
+> [!WARNING]
+> Flatpak package is not maintained by us. Due to that we cannot recommend using it as as there maybe some unexpected issues.
+
+Flatpak by default restricts access to themes, to allow access, use the following: 
+```bash
+sudo flatpak override --filesystem=$HOME/.local/share/themes
+```
+
+After you've allowed access, set the theme, using the following:
+```bash
+# Change to suite your flavor / accent combination
+export FLAVOR="mocha"
+export ACCENT="mauve"
+
+# Set the catppuccin theme directory
+export THEME_DIR="catppuccin-${FLAVOR}-${ACCENT}-standard+default"
+
+# Set the theme
+sudo flatpak override --env=GTK_THEME=$THEME_DIR
```

Comment:
This does not make any sense to me, its not a themedir, so `$THEME_DIR` doesn't seem like a logical name. And adding `export FLAVOR="mocha"` seems overkill. I think its just easier to have
 ```bash
 sudo flatpak override --env=GTK_THEME="catppuccin-${FLAVOR}-${ACCENT}-standard+default"
 ```

Reply:
oh